### ISSUE TITLE
Rollup version is logged twice when clearScreen is set to false

### ISSUE THREAD

**Issue by vladshcherbin** [rollup/rollup#2314]
When `clearScreen` is set to `false`, rollup version is logged twice.

Rollup config:

```js
export default {
  input: 'src/index.js',
  output: {
    file: 'dist/app.js',
    format: 'cjs'
  },
  watch: {
    clearScreen: false
  }
}
```

Console

```bash
yarn rollup -c -w
```

Result

![screen shot 2018-07-02 at 09 19 15](https://user-images.githubusercontent.com/6711845/42148389-aa3ec58c-7dda-11e8-80ea-c8b3bf1bf411.png)

Rollup version: 0.62

Reproduction repo - https://github.com/vladshcherbin/experiments/tree/master/rollup-watch-twice

**Comment by vladshcherbin** [comment]
@gkatsev since you made a [recent fix](https://github.com/rollup/rollup/pull/2125) to `clearScreen`, maybe you know something about this 🕵️

**Comment by gkatsev** [comment]
My initial thought is that the first one gets printed when rollup starts and then the second one gets printed as part of the `clearScreen`, but because it doesn't get cleared, it gets printed right below the second one. However, that's just a hunch, I can try and take a look at some point.

**Comment by vladshcherbin** [comment]
@gkatsev I also think so, seems like it's printed first [here](https://github.com/rollup/rollup/blob/v0.62.0/bin/src/run/watch.ts#L94) and then [here](https://github.com/rollup/rollup/blob/v0.62.0/bin/src/run/watch.ts#L114)

**Comment by gkatsev** [comment]
looks like the `start` method should have access to the `clearScreen` variable, we probably want to skip one of those prints if `clearScreen` is set to false. Probably the one in the `START` event handler.

**Comment by vladshcherbin** [comment]
Fixed and shipped in 0.63 🎉
